### ISSUE TITLE
Use news items to describe whats new

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/portal#138]
On every release we should create a news item that describes what is new in this monthly or annual version.
This will replace the whats new stuff o the metadata page and allows the editor to maintain a draft post in the portal while he is editing content. For annual releases the whats new should be aggregated from all monthly releases which can easily be done by checking the last 11 monthly news items.

**Comment by mdoering** [comment]
@yroskov @gdower @chantalhuijbers would you mind if we try this already for the current April version?
I would create a news item based on the current metadata and you can then continue with that as a template for future posts.

**Comment by yroskov** [comment]
Yes, let's try.

I am interested in a template as a part of **metadata** for filling lists of new and updated GSDs, plus occasional paragraph for "other improvements" or "known problems". It's all about content only.  

These data might be used in the portal, but they also must stay and be archived in metadata of each edition.

**Comment by mdoering** [comment]
They will stay with stable URLs in the portal so that they can be linked from any release also from ChecklistBank, e.g. https://www.catalogueoflife.org/2021/04/14/stable-ids

**Comment by yroskov** [comment]
Will these info go in export files as well?

**Comment by mdoering** [comment]
No. At least can't see an easy way for that to happen. They also never did in the old DwC-A exports.

Alternatively I had considered to add a `changes` field to the metadata. We could even automatically aggregate the changes from all monthly releases for the annual one then. BUT it would mean we just have unstyled or very simply styled paragraphs. No lists, tables, images, etc.

**Comment by mdoering** [comment]
deployed to preview: https://preview.catalogueoflife.org/2021/04/05/release

**Comment by mdoering** [comment]
@yroskov @chantalhuijbers if you are happy with that I would deploy it also to prod. I would prefer this to the previous approach.
